### ISSUE TITLE
Review the alt text for Communities of Practice page images and ensure that they adhere to WCAG

### ISSUE THREAD

**Issue by IsaacDesigns** [hackforla/website#2571]
### Overview
We need to ensure that alt text for images adhere to Web Content Accessibility Guidelines (WCAG). For this issue, we will tackle the images under the Communities of Practice page.

#### Details
Developers identified that some alt texts in the Communities of Practice page should be reviewed because they are either inaccurate or too long. Ideally, alt text should be descriptive but short and succinct. According to the WCAG page, an accompanying alt text should be short. Additionally, the Functional Accessibility Evaluator (FAE) recommends for alt text to be no more than 100 characters.

### Action Items
- [x] Make a copy of the tab Template [Page Name]- [Status] 
- [x] With the copied tab replace [Page Name] with the page you are reviewing and [Status] with "Audit in Progress". Don't forget to remove brackets "[ ]"
- [x] Add a link of the page your working on to the Resources/Instructions section of this issue at the bottom where it says "Google sheet Tab for your page".
- [x] For each image on the Communities of Practice page identify if the alt text needs to be changed, or not, using the recourses below, and fill out the google sheet. To locate the alt text use the Google Chrome browser, right click "inspect" to view pages code.
- [x] After completing auditing each image change your google sheet tabs [Status] to Audit Complete.
- [x] Review with Design Lead or another member of the team
- [x] Mark spreadsheet tab as QA (e.g., About Page - Audit QA)
- [x] Add note at bottom of issue saying its ready for QA and use the close with comment, so that it goes to the QA column

Product
- [x] Review the work and sign off on it if ready
   - [x] Mark spreadsheet tab as Complete (e.g., About Page - QA Complete)
   - [x] Add Ready for dev leads label
   - [x] Move to New Issue Approval Column

Development
- [ ] Create issues for that particular alt text should be created (preferably labelled good first issue) for new developers to fix
- [ ] Close this issue

### Resources/Instructions
[Communities of Practice page](https://www.hackforla.org/communities-of-practice)
[Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
[Decorative Images and alt text](https://www.w3.org/WAI/tutorials/images/decorative/)
[WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
[FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)
[Google sheet instructions](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=308160713)
[Google sheet template](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=129703697)
[Google sheet Tab for Communities of Practice](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=703949441)

**Comment by jenjen26** [comment]
Progress: waiting for team review
Blockers: none 
Availability: 
ETA: complete

**Comment by IsaacDesigns** [comment]
I've participated to review this issue and close it if it is done.

**Comment by IsaacDesigns** [comment]
@jenjen26 in the "Area/ Section in Code" column it looks like you put the actual code and I cannot locate what image you are referring to with this code. Please revise the "Area/ Section in Code" column with appropriate names. For examples see the tabs of other google sheet page's "Area/ Section in Code" column. 

So others don't make this mistake I've changed "Area/ Section in Code" to "Area on Page".

**Comment by jenjen26** [comment]
@IsaacDesigns I edited and it's ready for review.

**Comment by IsaacDesigns** [comment]
@jenjen26 thanks for updating the issue!

**Comment by 7kram** [comment]
I QAed this sheet and it's now ready for dev leads to make good first and second issues out of it.

**Comment by ExperimentsInHonesty** [comment]
I agree its ready for dev leads to make issues

**Comment by ExperimentsInHonesty** [comment]
https://github.com/hackforla/website/blob/gh-pages/pages/communities-of-practice.html

We only need to revise this line from
```
alt="{{ community[1].alt }}"
```
to
```
alt=""
```

**Comment by MattPereira** [comment]
Addressing the [comment above](https://github.com/hackforla/website/issues/2571#issuecomment-1399579073) with the following issue creation
- #3860
